src/style/main.scss

```diff
@@ -1,3 +1,8 @@
+html {
+    font-size: 62.5%;
+}
```

Comment:
Is this supposed to be here?

Reply:
Its part of support for EM for fonts which i started to bring in from Rodys branch

Reply:
needs to be in the `.swagger-ui` block - not in `html`